src/SprykerEco/Shared/Minubo/MinuboConstants.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Minubo;
+
+interface MinuboConstants
+{
+    const MINUBO_FILE_SYSTEM_NAME = 'MINUBO:MINUBO_FILE_SYSTEM_NAME';
```

Comment:
New PHP7 demands a visibility.
Is this protected or public?

Reply:
👍